react/components/AvailabilitySubscriber/index.js

```diff
@@ -1,14 +1,49 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { ApolloConsumer } from 'react-apollo'
-import { injectIntl, intlShape } from 'react-intl'
+import { injectIntl, intlShape, defineMessages } from 'react-intl'
 
 import { Button, Input } from 'vtex.styleguide'
 
 import ADD_TO_AVAILABILITY_SUBSCRIBER_MUTATION from './mutations/addToAvailabilitySubscriberMutation.gql'
 
 import styles from './styles.css'
 
+const messages = defineMessages({
+  availabilitysubscriberInvalidemail: {
+    id: 'store/availability-subscriber.invalid-email',
+    from: 'vtex.store-messages'
+  },
+  availabilitysubscriberAddedmessage: {
```

Comment:
a lot of these messages have capitalCase naming problems...

Reply:
They follow the convention of only capitalizing in the `.` and not in the `-`.

Reply:
I've just made this pr camel case compliant